recipes/sklearn-contrib-lightning/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{%set name = "sklearn-contrib-lightning" %}
+{%set version = "0.2.0" %}
+{%set hash_type = "md5" %}
+{%set hash_val = "cf373cf22e9c1b0c11133589d3b14926" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
```

Comment:
Add a build section here like:

```yaml
build:
    number: 0
    script: python setup.py install --single-version-externally-managed --record record.txt
```

The `build/script` will do the job of the `build.sh` and `bld.bat`. The extra options are recommend when using setuptools to produce a package that plays nice with conda.